includes/api/class-wc-rest-product-categories-controller.php

```diff
@@ -126,7 +126,7 @@ protected function update_term_meta_fields( $term, $request ) {
 
 				$image_id = wc_rest_set_uploaded_image_as_attachment( $upload );
 			} else {
-				$image_id = absint( $request['image']['id'] );
+				$image_id = isset( $request['image']['id'] ) ? absint( $request['image']['id'] ) : 0;
```

Comment:
@claudiosanches $image_id with 0 will not pass the next if statement. And than the id will not be updated/deleted.